war/src/main/webapp/scripts/prototype.js

```diff
@@ -908,8 +908,8 @@ var Enumerable = (function() {
 
   function findAll(iterator, context) {
     var results = [];
-    this.each(function(value, index) {
-      if (iterator.call(context, value, index))
+    this.forEach(function(value, index, self) {
```

Comment:
Tippy would error if I didn't change/comment this out - not too sure if this is the best solution/if theres a workaround